playbooks/k8s-cluster.yml

```diff
@@ -187,3 +187,16 @@
       delegate_to: localhost
   tags:
     - local
+
+- name: Add Mellanox RoCE backend
+  import_playbook: roce/roce.yaml
+  vars:
+    pf1: ens9f0
```

Comment:
These vars should probably go in a config file so a user could modify them

Reply:
Sure. We are working on creating one configuration file to all. Want to synchronize with your team.